hw/dv/tools/dvsim/sim.mk

```diff
@@ -20,22 +20,28 @@ prep_tool_srcs:
 	${LOCK_TOOL_SRCS_DIR} "cp -Ru ${tool_srcs} ${tool_srcs_dir}/."
 
 pre_compile: prep_tool_srcs
-	@echo "[make]: pre_compile"
+	@echo "[make]: pre_build"
```

Comment:
target name and print log aren't matching. perhaps change both
same for line 34, 38, 44.

Reply:
Updated, thanks.